website/content/docs/concepts/integratedstorage/autopilot.mdx

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+page_title: Integrated Storage Autopilot
+sidebar_title: Autopilot
+description: Learn about the autopilot subsystem of integrated raft storage in Vault.
+---
+
+# Autopilot
+
+Autopilot enables automated workflows for managing Raft clusters. The current
+feature set includes 3 main features: Server Stabilization, Dead Server Cleanup
+and State API. These three features are introduced in Vault 1.7.
+
+## Server Stabilization
+
+Server stabilization helps to retain the stability of the Raft cluster by safely
+joining new voting nodes to the cluster. When a new voter node is joined to an
+existing cluster, autopilot adds it as a non-voter instead, and waits for a
+pre-configured amount of time to monitor it's health. If the node remains to be
+healthy for the entire duration of stabilization, then that node will be
+promoted as a voter. The server stabilization period can be tuned using
+`server_stabilization_time` (see below).
+
+## Dead Server Cleanup
+
+Dead server cleanup automatically removes those nodes from the Raft, that are
+deemed failed, avoiding the manual operator intervention. This feature can be
```

Comment:
```suggestion
Dead server cleanup automatically removes nodes deemed unhealthy from the
Raft cluster, avoiding the manual operator intervention. This feature can be
```

Reply:
We don't remove nodes that are deemed unhealthy. We only mark those as unhealthy. We only remove those that are deemed failed/dead (via a dedicated threshold).